examples-standalone/grid-charts-integration/src/app/app.module.ts

```diff
@@ -35,18 +40,22 @@ import 'hammerjs';
         DropDownsModule,
         PopupModule,
         ContextMenuModule,
-        InputsModule
+        InputsModule,
+        NotificationModule
```

Comment:
The NotificationModule can be removed if using the Notification service will not be required.